scripts/push_version.sh

```diff
@@ -37,6 +37,6 @@ function create_pull_request() {
   gh pr create --reviewer danschultz --title "[$(git_current_branch | tr '[:lower:]' '[:upper:]')]: Update to latest spec"
 }
 
-generate_client
-version_package "$@"
+# generate_client
+# version_package "$@"
```

Comment:
Do you want to merge this?

Reply:
Definitely not, missed this in my last commit after running into some problems with the `create_pull_request` step 🙃. Thanks for pointing that out - this should be fixed!

Reply:
Definitely not, missed this in my last commit after running into some problems with the `create_pull_request` step 🙃. Thanks for pointing that out - this should be fixed!